lib/widget/containers/ventilation_guide_container.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+import '../../models/ventilation_guide.dart';
+import '../../style.dart';
+import '../../widget/lists/ventilation_guide_card.dart';
+
+class VentilationGuideContainer extends StatelessWidget {
+  final List<VentiliationGuideSection> guide;
```

Comment:
would be nice to rename the `guide` prop to make it obvious its actually a list of guide sections